src/styles/blocks/veriables.scss

```diff
@@ -0,0 +1,4 @@
+$width-of-items: 100%;
+$font-size-12: 12px;
+$color-grey: rgb(97, 96, 112);
```

Comment:
It's better to use HEX color codes instead of RGB for consistency and readability. For example, the RGB color (97, 96, 112) can be represented as #616070 in HEX.